src/core/api.proto

```diff
@@ -30,6 +30,47 @@ package nvidia.inferenceserver;
 
 //@@.. cpp:namespace:: nvidia::inferenceserver
 
+//@@      .. cpp:var:: message InferSharedMemoryElement
```

Comment:
Comment indenting (spaces between //@@ and text) is wrong here. Look at an existing message to see how it should be at each level of your new messages. We have to be precise about this because of the way the doc generator works for protobuf files.

Reply:
Fixed